ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.component.js

```diff
@@ -73,40 +69,28 @@ export default class ConfirmEncryptionPublicKey extends Component {
 
   renderBalance = () => {
     const {
-      conversionRate,
       nativeCurrency,
-      currentCurrency,
       fromAccount: { balance },
     } = this.props;
     const { t } = this.context;
 
-    const nativeCurrencyBalance = conversionRate
-      ? formatCurrency(
-          getValueFromWeiHex({
-            value: balance,
-            fromCurrency: nativeCurrency,
-            toCurrency: currentCurrency,
-            conversionRate,
-            numberOfDecimals: 6,
-            toDenomination: EtherDenomination.ETH,
-          }),
-          currentCurrency,
-        )
-      : new Numeric(balance, 16, EtherDenomination.WEI)
-          .toDenomination(EtherDenomination.ETH)
-          .round(6)
-          .toBase(10)
-          .toString();
+    const nativeCurrencyBalance = new Numeric(
```

Comment:
Any idea how we can test that these changes are correct?  Not sure ive used encrypt/decrypt feature

Reply:
If you go to [test-dapp](https://metamask.github.io/test-dapp/); then look for this; that should be the one 🙏 
<img width="728" alt="Screenshot 2024-09-25 at 00 32 39" src="https://github.com/user-attachments/assets/a84e4452-ca64-43b9-8aea-8c0016de223a">